### PR TITLE
(fixes #158)(fixes #120)(fixes #122) restyle content columns

### LINKED ISSUE #120: de-emphasize the sidebar
This has no concrete definition and will likely result in some back-and-forth discussion. A good first try would be to lessen the background-color opacity like `background: rgba(255,255,255, 0.3)` and border lines.

### LINKED ISSUE #122: consider more visual emphasis on content area
This is quite loose and will likely be less styling work than it is discussion. Consider adding a border, border-radius, box-shadow etc.

### LINKED ISSUE #158: change description column's detail-heading--01 to bolder
Bump from 100 to 200

### PR DESCRIPTION
**Pull request by spasovski**
It's probably better to pull this locally for preview - I didn't want to screenshot too much data even if it isn't real. This is obviously one attempt - we may want to iterate a few times until we agree on a strategy.

image

### CODE REVIEW

File: src/app/sections/GraphicBody.svelte [hunk 1]
```diff
@@ -126,6 +125,8 @@ function handleBodySelectors(event) {
   background-color: white;
   padding: var(--space-4x);
   padding-top: var(--space-2x);
+  border-radius: var(--border-radius-1h);
+  box-shadow: 0 0 3px 2px rgba(0,0,0,0.1) inset;
```
Comment (hamilton) [inline src/app/sections/GraphicBody.svelte]:
```suggestion
  border: 1px solid var(--cool-gray-100);
```

Comment (hamilton) [reply src/app/sections/GraphicBody.svelte]: The inset shadow is kind of intense and might be a bit distracting. I'd maybe recommend:

1. removing that,
2. adding a 1px solid border to the body element

**Review by hamilton — CHANGES_REQUESTED** [review]
Thanks for the PR! I agree that we'll probably have to have some back-and-forth on this. Here are some thoughts on where it is now:

1. I love the bg color to the sidebar – definitely helps separate that bit of content adn de-prioritizes it.
2. the inset feels a bit intense, I think. We might be able to accomplish a bit of separation by having a 1px solid right border on the  body.
3. it seems like it might help if we have the details + body in its own element + apply a light box shadow to that instead of the grid approach we have here. That'll help with keeping the body content as a whole the most important element on the screen, and still maintain the de-prioritized details.

Let me know what you think ~

**Comment by spasovski** [comment]
@hamilton re-r? Likely needs another few iterations of comments but no big deal.

**Review by hamilton — CHANGES_REQUESTED** [review]
Thanks for the tweaks. A few comments that we can talk about more tomorrow:

- the depth variable on the shadow seems to only apply to `graphic-body__content`. I think the solution here is to have a parent element that encompasses `graphic-body__content` and `graphic-body__details` (as well as another grid for that new parent element) and you can apply the shadow variable to that. As it is now the difference between `--subtle-cool` and the bg color is pretty minimal, and we aren't achieving any depth on the right side there. I'm open to changing the feel of the depth variable if it feels "too deep", but we should probably get the styling there consistent either way.
- It does still feel that maybe a 2px solid border b/t the body & the right column might help separate things a bit. I found `--cool-gray-100` to look pretty good. Worth playing with.
- it definitely feels like the top two lines (histogram nightly active ###) could be a different color, or maybe a slight linear gradient like `linear-gradient(to top right, white 50%, var(--cool-gray-100))`. Not sure, but as it is now the label element you created doesn't feel like it works well against the background.

**Comment by spasovski** [comment]
@hamilton should be ready for next round of review

- adjusted a11y contrast of StatusLabel states
- added letter-spacing + font-weight: 200 on detail-header__01

**Comment by hamilton** [comment]
@spasovski thanks for the changes. This is looking great. I am noticing that the spacing you added for the header creates a scrollbar where there wasn't one before. Let's either fix that or reverse the change and address it later. Should be a quick fix either way – once it's done, feel free to merge this.

**Comment by spasovski** [comment]
@hamilton awesome thanks and yea I'll just undo it and have it go in its own PR

**Pull request merged**